### PR TITLE
CPDP-2710 net gain guidance

### PR DESCRIPTION
**Pull request by nubz**

### CODE REVIEW

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/FurtherReturnGuidanceControllerSpec.scala [hunk 1]
```diff
@@ -651,6 +651,66 @@ class FurtherReturnGuidanceControllerSpec
...
+
+          def test(
+            individualUserType: IndividualUserType,
+            userType: UserType
+          ): Unit = {
+            inSequence {
+              mockAuthWithNoRetrievals()
+              mockGetSession(
+                sessionDataWithFillingOutReturnForSingleDisposals(
+                  IncompleteSingleDisposalTriageAnswers.empty.copy(
```
Comment (rambabu-posa) [inline test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/FurtherReturnGuidanceControllerSpec.scala:676]: Please add unit tests for Multiple Disposals too and also cover other journey like "StartingNewDraftReturn and StartingAmendReturn"

Comment (nubz) [reply test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/FurtherReturnGuidanceControllerSpec.scala:676]: problem is startingNewDraftReturn and startingAmendReturn is those states only apply to returns that have not yet been saved as a draft - it is only possible to get to the net gain page in section 3 whilst filling out a saved draft return (after triage is complete then draft is saved and other sections can be accessed) - also the net gain page and guidance page are the same with no variations for single or multiple - but I can add the same test for multiple answers anyway just to be sure...

Comment (nubz) [reply test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/FurtherReturnGuidanceControllerSpec.scala:676]: added multiple case

**Comment by rambabu-posa** [comment]
LGTM

**Pull request merged**